drivers/power/adp1050/adp1050.h

```diff
@@ -106,13 +117,22 @@
 
 #define ADP1050_OUTA_OL_ENABLE			NO_OS_BIT(0)
 #define ADP1050_OUTB_OL_ENABLE			NO_OS_BIT(1)
+#define ADP1051_OUTC_OL_ENABLE			NO_OS_BIT(2)
+#define ADP1051_OUTD_OL_ENABLE			NO_OS_BIT(3)
 #define ADP1050_SR1_OL_ENABLE			NO_OS_BIT(4)
 #define ADP1050_SR2_OL_ENABLE			NO_OS_BIT(5)
 
-#define ADP1050_OUTB_SR2_RISING_MOD_MASK	NO_OS_GENMASK(7, 6)
-#define ADP1050_OUTB_SR2_FALLING_MOD_MASK	NO_OS_GENMASK(5, 4)
-#define ADP1050_OUTA_SR1_RISING_MOD_MASK	NO_OS_GENMASK(3, 2)
-#define ADP1050_OUTA_SR1_FALLING_MOD_MASK	NO_OS_GENMASK(1, 0)
+#define ADP1050_OUTB_OUTD_SR2_RISING_MOD_MASK	NO_OS_GENMASK(7, 6)
+#define ADP1051_OUTD_SR2_RISING_MOD_MASK	NO_OS_GENMASK(7, 6)
+
+#define ADP1050_OUTB_OUTD_SR2_FALLING_MOD_MASK	NO_OS_GENMASK(5, 4)
+#define ADP1051_OUTD_SR2_FALLING_MOD_MASK	NO_OS_GENMASK(5 ,4)
+
+#define ADP1050_OUTA_OUTC_SR1_RISING_MOD_MASK	NO_OS_GENMASK(3, 2)
+#define ADP1051_OUTC_SR1_RISING_MOD_MASK	NO_OS_GENMASK(3, 2)
+
+#define ADP1050_OUTA_OUTC_SR1_FALLING_MOD_MASK	NO_OS_GENMASK(1, 0)
+#define ADP1051_OUTC_SR1_FALLING_MOD_MASK	NO_OS_GENMASK(1, 0)
```

Comment:
I'm guessing that this is similar to my previous comments. Check if you can have a more scalable way in doing this.